### PR TITLE
[Jenkins] When running tests on VSTS do specify a device type.

### PR DESCRIPTION
**Pull request by mandel-macaque**
We are going to split the work between different types of devices:

- iOS
- iOS Beta
- AppleTv
- AppleTv Beta

So the idea is that before we have the beta devices and the split execution, we should be able to pass the device type so that when the person doing the monitoring can state which of the devices failed.

We add a default value so that the device test works.

### CODE REVIEW

File: jenkins/vsts-device-tests-set-status.sh [hunk 1]
```diff
@@ -30,6 +39,11 @@ if test -z "$TOKEN"; then
 	exit 1
 fi
 
+if test -z "$DEVICE_TYPE"; then
+	# set a default value
+	DEVICE_TYPE="iOS/AppleTv"
```
Comment (rolfbjarne) [inline jenkins/vsts-device-tests-set-status.sh]: `iOS/tvOS` sounds better, since those are the official platform names.

**Comment by mandel-macaque** [comment]
Failures can be ignored.

**Comment by mandel-macaque** [comment]
@monojenkins backport to d16-1

**Comment by mandel-macaque** [comment]
@monojenkins backport to 10.2

**Pull request merged**